app/lib/url.rb

```diff
@@ -70,4 +70,15 @@ def self.local_image(image_name, host: nil)
   def self.organization(organization)
     url(organization.slug)
   end
+
+  # Ensures we don't consider an empty referer
```

Comment:
Sorry if my original comment was unclear, but we no longer need this due to the other 2 changes. They were the only two call sites of this method and I think now that we no longer have the service worker guard clause this method doesn't add much.